cEP-0001.md

```diff
@@ -4,38 +4,101 @@ Decision Making in coala
 |Metadata|                                                                    |
 |--------|--------------------------------------------------------------------|
 |cEP     |0001                                                                |
-|Version |1.1                                                                 |
+|Version |1.2                                                                 |
```

Comment:
I agree with sils here, it seems more like this entire document has been restructured to describe what the different roles can do, their voting rights, and an outline of the voting types, rather than just an outline of the voting types.

Thoughts on this?